lib/src/repo.rs

```diff
@@ -1730,6 +1730,27 @@ impl MutableRepo {
         view.set_tag_target(name, new_target);
     }
 
+    pub fn get_remote_tag(&self, symbol: RemoteRefSymbol<'_>) -> RemoteRef {
```

Comment:
Can the anonymous lifetime be elided here an below? I think they're sometimes needed in async functions these days but I may misremember. Are they recommended in general now? Or is it actually needed here for some reason?